ui/raidboss/data/05-shb/ultimate/the_epic_of_alexander.js

```diff
@@ -198,10 +200,10 @@ let getHeadmarkerId = (data, matches) => {
       infoText: {
         en: 'Hide Behind Ice',
         de: 'Hinter dem Eis verstecken',
+        fr: 'Cachez-vous derrière la glace',
```

Comment:
I try to translate as faithfully as possible, besides I have a file with the expressions to stay on the same translations. But I also take the conjugation into account and in French, we put that in the imperative, because we start from the principle that it is an order to be respected absolutely.